Dockerfile

```diff
@@ -6,16 +6,18 @@ RUN mkdir -p /linode
 WORKDIR /linode
 
 ARG VERSION
+ARG TARGETARCH
+ARG TARGETOS
 
 COPY go.mod .
 COPY go.sum .
 COPY main.go .
 #COPY pkg ./pkg
 
 RUN go mod download
-RUN go build -a -ldflags '-X main.version='${VERSION}' -extldflags "-static"' -o /bin/k8s-node-decorator /linode
+RUN GOARCH=${TARGETARCH} GOOS=${TARGETOS} go build -a -ldflags '-X main.version='${VERSION}' -extldflags "-static"' -o /bin/k8s-node-decorator /linode
```

Comment:
you sure we need this?

Reply:
I just removed it and it works well. Thanks!